index.md

```diff
@@ -124,6 +124,14 @@ See [opensearch.org/javadocs/](https://opensearch.org/javadocs/).
 
 The project welcomes GitHub issues, bug fixes, features, plugins, documentation---anything at all. To get involved, see [Contributing](https://opensearch.org/source.html) on the OpenSearch website.
 
+## Test experimental features
```

Comment:
I would explicitly add "We do not recommend use of experimental features in production."